core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -48,7 +48,16 @@ case class RawlsBillingProject(projectName: RawlsBillingProjectName, status: Cre
   // def instead of val because val confuses the json formatter
   def googleProjectId: GoogleProjectId = GoogleProjectId(projectName.value)
 }
-case class RawlsBillingProjectResponse(projectName: RawlsBillingProjectName, billingAccount: Option[RawlsBillingAccountName], servicePerimeter: Option[ServicePerimeterName], invalidBillingAccount: Boolean, roles: Set[ProjectRoles.ProjectRole])
+
+case class WorkspaceBillingAccount(workspaceName: WorkspaceName, maybeBillingAccountName: Option[RawlsBillingAccountName])
```

Comment:
in a response object don't call it `maybeBillingAccountName` as the `maybe` looks weird in json. And if it is `None` it won't be in the json at all.